src/styles/typography.js

```diff
@@ -8,53 +8,11 @@ const baseTextStyle = {
   MozOsxFontSmoothing: "grayscale"
 };
 
-const baseListStyle = {
-  width: "100%",
-  marginLeft: 0,
-  paddingLeft: "2rem"
-};
-
 // Helpers
 
 // Modular scale font size helper; level can be negative (for smaller font sizes) and positive (for larger font sizes) integers; level 0 === baseFontSize
 export const getFontSize = ({ baseFontSize, msRatio }, level = 0) =>
-  `${baseFontSize * Math.pow(msRatio, level)}px`;
-
-const inlineElements = (theme, selector = "") => {
-  return {
-    [`${selector} i, em`]: {
-      fontStyle: "italic"
-    },
-    [`${selector} b, strong`]: {
-      fontWeight: 700
-    },
-    [`${selector} a, a:visited`]: {
```

Comment:
In the new `Content/Markdown.js` we no longer handle `a:visited` from what I can tell, but we should.

Reply:
It doesn't seem necessary – at least in Chrome and FF 🤔

Also, these selectors didn't even work properly! They should've been `${selector} a, ${selector} a:visited` etc.